### ISSUE TITLE
Zoo Workflow Upgrade: Plot Reference Shoreline Buffer on the Detection Figure

### ISSUE THREAD

**Issue by 2320sharon** [Doodleverse/CoastSeg#210]
In the zoo workflow the diagram showing the detection of the extracted Shoreline figures does not show the reference Shoreline buffer like the coast at workflow does. Add in the necessary code to show the reference Shoreline buffer on the Shoreline detection images

**Comment by 2320sharon** [comment]
@dbuscombe-usgs 
I also added the fix the minimize the amount of white space per figure by using `bbox_inches="tight"` within the `save_detection_figure` function. I added the same code to the coastsat_package 0.1.43 , so that the shoreline detection figures generated by the coastsat workflow also minimize the amount of white space per figure.
```
    fig.savefig(
        os.path.join(filepath, date + "_" + satname + ".jpg"),
        dpi=150,
        bbox_inches="tight",
    )
```

**Comment by 2320sharon** [comment]
Closed in version 1.1.27
